resources/views/layouts/sidebars/app.blade.php

```diff
@@ -82,6 +85,9 @@
                                     {!! $child['custom_label'] ?: $child['label'] !!}
                                 </a>
                             </li>
+                            @if($currentCampaign->boosted())
+                                @include('layouts.sidebars._quick-links')
```

Comment:
This needs to be refactored to only do the query a single time per sidebar rendering :) 
![image](https://user-images.githubusercontent.com/3263440/188280189-6d76739b-d4e6-4652-a95a-a79fe15db339.png)